.github/workflows/backend-ci.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
```

Comment:
Should update comment to remove mention of linting, as we do not currently lint in this workflow.